app/src/main/java/io/intrepid/contest/screens/contestjudging/scoreentrydetail/ScoreEntryDetailPresenter.java

```diff
@@ -0,0 +1,63 @@
+package io.intrepid.contest.screens.contestjudging.scoreentrydetail;
+
+import android.support.annotation.NonNull;
+
+import java.util.List;
+
+import io.intrepid.contest.R;
+import io.intrepid.contest.base.BasePresenter;
+import io.intrepid.contest.base.PresenterConfiguration;
+import io.intrepid.contest.models.Category;
+import io.intrepid.contest.models.Entry;
+import io.intrepid.contest.rest.ContestWrapper;
+import io.reactivex.disposables.Disposable;
+import io.reactivex.functions.Consumer;
+import timber.log.Timber;
+
+class ScoreEntryDetailPresenter extends BasePresenter<ScoreEntryDetailContract.View> implements ScoreEntryDetailContract.Presenter, Consumer<ContestWrapper> {
+
+    private final Entry entry;
+    private final String fragmentTitle;
+    private List<Category> categories;
+
+    ScoreEntryDetailPresenter(@NonNull ScoreEntryDetailContract.View view,
+                              @NonNull PresenterConfiguration configuration,
+                              Entry entry,
+                              String fragmentTitle) {
+        super(view, configuration);
+        this.entry = entry;
+        this.fragmentTitle = fragmentTitle;
+    }
+
+    @Override
+    public void onViewCreated() {
+        super.onViewCreated();
+        getActiveContest();
+        view.showFragment(entry, fragmentTitle);
```

Comment:
I may still change this to the ViewPager impl, which will use the viewpager activity layout instead so working with it but expecting it to not necessarily stay the same.